src/main/scala/io/gitlab/agluszak/tickets/http/requests/ReservationCreateRequest.scala

```diff
@@ -0,0 +1,6 @@
+package io.gitlab.agluszak.tickets.http.requests
+
+import io.gitlab.agluszak.tickets.{ScreeningId, TicketsQuantity}
+
+case class ReservationCreateRequest(screeningId: ScreeningId, name: String,
+                                    surname: String, seats: List[Int], tickets: List[TicketsQuantity])
```

Comment:
From purely scala point of view I could use NEL https://typelevel.org/cats/datatypes/nel.html
I was thinking about merging the seat numbers list and tickets list into one, so it would be impossible to submit a request containing more seats than tickets or the other way round, but on the other hand nothing was said about assigning tickets to specific places. But on the other other hand they could be assigned arbitrarily (by frontend) and we wouldn't need to check that (because who cares if a parent and their child switched seats). So maybe that was a better solution after all.